README.md

```diff
@@ -22,26 +41,183 @@ allprojects {
 }
 ```
 
-2. Add your dependency to your app's `build.gradle` file:
+### Available Artifiacts
+
+2. Add relevant dependencies to your app's `build.gradle` file:
 
 ```groovy
-implementation 'io.customer.android:sdk:<version-here>'
+implementation 'io.customer.android:tracking:<version-here>'
+implementation 'io.customer.android:messaging-push-fcm:<version-here>'
 ```
 
-Replace `version-here` with the the latest version: ![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.customer.android/sdk/badge.svg)
+Replace `version-here` with the the latest version: ![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.customer.android/tracking/badge.svg)
 
-# Documentation
+# Initialize the SDK
 
-...Not yet ready. Under development. 
+Before you can use the Customer.io SDK, you need to initialize it. `CustomerIO` is a singleton: you'll create it once and re-use it across your application.
 
-# Contributing
+```kotlin
+class App : Application() {
+   override fun onCreate() {
+       super.onCreate()
+      val customerIO = CustomerIO.Builder(
+            siteId = "your-site-id",
+            apiKey = "your-api-key",
+            appContext = this
+        ).build()
+   }
+}
+```
+The Builder for CustomerIO exposes configuration options for features such `region`,
+`timeout`. 
+
+> A best practice is to initialize CustomerIO in the Application class, this way you will be able to access that instance from any part of your application using the instance() method:
+
+# Tracking
+
+### Dependency
+Tracking artifact is needed in order to use tracking features
+```groovy
+implementation 'io.customer.android:tracking:<version-here>'
+```
+## Identify a customer
+
+When you identify a customer, you:
+1. Add or update the customer's profile in your workspace.
+2. Save the customer's information on the device. Future calls to the SDK are linked to the last-identified customer. For example, after you identify a person, any events that you track are automatically associated with that person.
+
+You can only identify one customer at a time. The SDK "remembers" the most recently-identified customer.
+If you identify customer "A", and then call the identify function for customer "B", the SDK "forgets" customer "A" and assumes that customer "B" is the current app user. 
+
+[Learn more about the `Tracking` class][s-tracking].
+
+```kotlin
+CustomerIO.instance()
+            .identify(
+                identifier = "989388339",
+                attributes = mapOf("first_name" to "firstName")
+            ).enqueue { result ->
```

Comment:
I think it would be a good idea to introduce the design of the SDK in using `enqueue` and other functions of the `Action` class so customers can properly use the public functions.